### DISCUSSION TITLE
Problem with getting github student pack

### DISCUSSION THREAD

**Discussion by Yasir-090** [community/community#113686] [GitHub Education]
### Hi everyone,

I have just tried to get student pack of github. But I face an issue. It's says that your uploaded proof is already  used. Instead I can't used it before. On my first try it's said that I have to fix some issues before apply to that like 2 step factor etc. I solved all those problem and after that now when I apply it's shows this message. Anyone now how to solve this.
![github](https://github.com/community/community/assets/164096373/a137f246-2afe-4d89-9fd3-8c33c5eb0dad)

**Comment by samus-aran** [comment]
Hi @Yasir-090 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [Your uploaded proof has already been used previously.](https://github.com/orgs/community/discussions/111352#uploaded-proof-already-used)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍

**Comment by thilinaabey** [comment]
cancel current application and start new one
